lib/text.rb

```diff
@@ -0,0 +1,29 @@
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness
+require 'twilio-ruby'
+require 'dotenv'
+
+class Text
+
+def initialize
+  # put your own credentials here
+  account_sid = ENV["SID"]
+  auth_token = ENV["TOKEN"]
+
+  # set up a client to talk to the Twilio REST API
+  @client = Twilio::REST::Client.new account_sid, auth_token
+end
+
+
+def messege
+  @client.account.messages.create({
+	:from => ENV["TWILIONUM"],
+	:to => ENV["CLIENTNUM"],
+	:body => "Thank you! Your order was placed and will be delivered before #{time}",
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)